docs/README.md

```diff
@@ -45,7 +45,7 @@ Links:
 Once the entity and schema mapping have been finalized, you prepare the artifacts.  This includes:
 
 1. StatisticalVariable MCF nodes (if any) checked into [schema repo](https://github.com/datacommonsorg/schema/tree/main/stat_vars)
-2. Template MCF and corresponding cleaned tabular files (typically CSV)
+2. Template MCF and corresponding cleaned tabular files (typically CSV). MCF and Template MCF nodes may be written by hand when there are only a handful in number. Otherwise, these nodes can be generated via scripts.
```

Comment:
"Like StatisticalVariable MCF, the Template MCF can also be hand-written or script-generated."